microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -20,30 +19,292 @@
     Lscpu,
     Lspci,
     Make,
+    Meson,
     Modprobe,
+    Ninja,
     Pidof,
+    Pip,
     Pkgconfig,
-    Rm,
-    Service,
+    Python,
     Tar,
     Timeout,
     Wget,
 )
-from lisa.util import (
-    LisaException,
-    MissingPackagesException,
-    SkippedException,
-    UnsupportedDistroException,
-)
+from lisa.util import LisaException, SkippedException, UnsupportedDistroException
 from lisa.util.constants import DEVICE_TYPE_SRIOV, SIGINT
 from microsoft.testsuites.dpdk.common import (
+    DependencyInstaller,
+    Downloader,
+    GitDownloader,
+    Installer,
+    OsPackageDependencies,
+    PackageManagerInstall,
+    TarDownloader,
     is_ubuntu_latest_or_prerelease,
     is_ubuntu_lts_version,
+    is_url_for_git_repo,
+    is_url_for_tarball,
+    unsupported_os_thrower,
 )
 
 PACKAGE_MANAGER_SOURCE = "package_manager"
 
 
+# declare package dependencies for package manager DPDK installation
+DPDK_PACKAGE_MANAGER_PACKAGES = DependencyInstaller(
+    requirements=[
+        OsPackageDependencies(
+            matcher=lambda x: isinstance(x, Debian),
+            packages=["dpdk", "dpdk-dev"],
+            stop_on_match=True,
+        ),
+        OsPackageDependencies(
+            matcher=lambda x: isinstance(x, Suse)
+            and float(x.information.release) == 15.5,
```

Comment:
how about `>=`?

Reply:
I didn't add this check, so I'm not touching it.